docker-compose.yml

```diff
@@ -96,3 +96,27 @@ services:
       - CASSANDRA_PORT=9042
       - ORIENTDB_HOST=orientdb
       - ORIENTDB_PORT=2424
+
+  release:
+    build:
+      cache_from:
+        - quillscala/sbt:latest
+      context: .
+      dockerfile: ./build/Dockerfile-sbt
+    image: quillscala/release
```

Comment:
new compose service `release` which actually the same as `sbt` and based on that image however without dependencies of databases since they aren't needed during release